appservice/src/deploy/localGitDeploy.ts

```diff
@@ -21,9 +21,10 @@ import { waitForDeploymentToComplete } from './waitForDeploymentToComplete';
 export async function localGitDeploy(client: SiteClient, fsPath: string): Promise<void> {
     const kuduClient: KuduClient = await getKuduClient(client);
     const publishCredentials: User = await client.getWebAppPublishCredential();
-    const remote: string = nonNullProp(publishCredentials, 'scmUri');
+    const remote: string = `https://${nonNullProp(publishCredentials, 'publishingUserName')}:${nonNullProp(publishCredentials, 'publishingPassword')}@${client.gitUrl}`;
```

Comment:
Have we been in contact with the App Service team to figure out why this changed in kudu lite and if it was intentional? Maybe we can get them to fix on their side?

Reply:
Check yo email soooonnnn

Reply:
I think I'm fine with this - looks like `client.gitUrl` will handle most of the weird stuff like custom domains, etc. that I mentioned offline.

I am curious if there is a way to connect with a temporary key instead of always using the main credentials. But if the old scmUri used these credentials, I'm fine keeping the behavior as-is for this release.